packages/main/src/Breadcrumbs.js

```diff
@@ -0,0 +1,534 @@
+import ItemNavigation from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import {
+	isSpace,
+	isShow,
+} from "@ui5/webcomponents-base/dist/Keys.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import ResizeHandler from "@ui5/webcomponents-base/dist/delegate/ResizeHandler.js";
+import NavigationMode from "@ui5/webcomponents-base/dist/types/NavigationMode.js";
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import BreadcrumbsDesign from "./types/BreadcrumbsDesign.js";
+import BreadcrumbsSeparatorStyle from "./types/BreadcrumbsSeparatorStyle.js";
+import BreadcrumbsItem from "./BreadcrumbsItem.js";
+import {
+	BREADCRUMBS_ARIA_LABEL,
+	BREADCRUMBS_OVERFLOW_ARIA_LABEL,
+	BREADCRUMBS_CANCEL_BUTTON,
+} from "./generated/i18n/i18n-defaults.js";
+import Link from "./Link.js";
+import Label from "./Label.js";
+import ResponsivePopover from "./ResponsivePopover.js";
+import Popover from "./Popover.js";
+import List from "./List.js";
+import StandardListItem from "./StandardListItem.js";
+import Icon from "./Icon.js";
+
+// Templates
+import BreadcrumbsTemplate from "./generated/templates/BreadcrumbsTemplate.lit.js";
+import BreadcrumbsPopoverTemplate from "./generated/templates/BreadcrumbsPopoverTemplate.lit.js";
+
+// Styles
+import breadcrumbsCss from "./generated/themes/Breadcrumbs.css.js";
+import breadcrumbsPopoverCss from "./generated/themes/BreadcrumbsPopover.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-breadcrumbs",
+	managedSlots: true,
+	languageAware: true,
+	slots: /** @lends sap.ui.webcomponents.main.Breadcrumbs.prototype */ {
+
+		/**
+		 * Defines the component items.
+		 *
+		 * <br><br>
+		 * <b>Note:</b> Use the <code>ui5-breadcrumbs-item</code> component to define the desired items.
+		 * @type {sap.ui.webcomponents.main.IBreadcrumbItem[]}
+		 * @slot items
+		 * @public
+		 */
+		"default": {
+			propertyName: "items",
+			type: HTMLElement,
+			invalidateOnChildChange: true,
+		},
+
+	},
+	properties: /** @lends  sap.ui.webcomponents.main.Breadcrumbs.prototype */ {
+
+		/**
+		 * Defines the visual indication and behavior of the breadcrumbs.
+		 * Available options are <code>Standard</code> (by default) and <code>NoCurrentPage</code>.
+		 * <br><br>
+		 * <b>Note:</b> The <code>Standard</code> breadcrumbs show the current page as the last item in the trail.
+		 * The last item contains only plain text and not a link.
+		 *
+		 * @type {BreadcrumbsDesign}
+		 * @defaultvalue "Standard"
+		 * @public
+		*/
+		design: {
+			type: BreadcrumbsDesign,
+			defaultValue: BreadcrumbsDesign.Standard,
+		},
+
+		/**
+		 * Determines the visual style of the separator between the breadcrumb items.
+		 *
```

Comment:
List the options in the description in this way:
```js
* <br><br>
* Available options are:
* <ul>
* <li><code>TopStart</code></li>
* <li><code>BottomCenter</code></li>
* <li><code>BottomEnd</code></li>
* </ul>
```

Reply:
Done